src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/collection/AllCollectionFilter.java

```diff
@@ -66,6 +67,7 @@ public List<BuiltCondition> getAll() {
 
   @Override
   public BuiltCondition get() {
-    throw new UnsupportedOperationException("For $all filter we always use getAll() method");
+    throw ErrorCode.SERVER_INTERNAL_ERROR.toApiException(
+        "For $all filter we always use getAll() method");
```

Comment:
Memory comes back, if we can not use get() for all filters, that means we need to rethink about the interface design. I will add a todo.